NewForce-at-Mountwest/bangazon-api-farfalle
# Cf employee controller

# Description

This is the employee controller, which should have functionality to post, update, and get single/all employees. The delete request is included for testing purposes, so that our tests don't fill the database with dummy data. To run a delete, you must include "?PeteyDeletey=True" at the end of the user url.


## Related Ticket(s)
6

## Steps to Test
Run tests, make sure they all pass. Run client, make sure you can see both single and all employees. Posting to postman should work, and when you get all employees after testing, you should only see the base 3.


# Checklist:
